src/uct/ib/dc/accel/dc_mlx5.h

```diff
@@ -24,19 +24,27 @@ typedef struct uct_dc_mlx5_iface {
     uct_rc_mlx5_iface_common_t          mlx5_common;
     uct_ud_mlx5_iface_common_t          ud_common;
     uct_ib_mlx5_txwq_t                  dci_wqs[UCT_DC_IFACE_MAX_DCIS];
-
 } uct_dc_mlx5_iface_t;
 
 
 typedef struct uct_dc_mlx5_ep {
     uct_dc_ep_t                         super;
+    uint8_t                             is_global;
```

Comment:
can we remove this and reuse some bit in uct_ib_mlx5_base_av_t?

Reply:
do we have a spare bit(s) there ?

Reply:
we should have